lms/static/scripts/ui-playground/components/EditAssignment.tsx

```diff
@@ -0,0 +1,348 @@
+import {
+  ArrowLeftIcon,
+  Button,
+  Card,
+  CardActions,
+  CardContent,
+  Checkbox,
+  LinkButton,
+} from '@hypothesis/frontend-shared/lib/next';
+import Library from '@hypothesis/frontend-shared/lib/pattern-library/components/Library';
+import classnames from 'classnames';
+import type { ComponentChildren } from 'preact';
+
+function PickerCard({
```

Comment:
Are you planning on sharing components between the ui-playground and the actual app in future, or intending to keep them separate?